mltsp/Flask/flask_app.py

```diff
@@ -41,7 +31,10 @@
 from sklearn.externals import joblib
 import numpy as np
 import yaml
-from flask.ext import stormpath
+if os.getenv("flask_testing") == "True":
```

Comment:
Thinking about this, MLTSP_DEBUG and DEBUG should probably be the same thing.  DEBUG_LOGIN ?